astroquery/gaia/core.py

```diff
@@ -911,5 +915,20 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def get_status_messages(self):
+        """Retrieve the messages to inform users about
+        the status of JWST TAP
+        """
+        try:
+            subContext = conf.GAIA_MESSAGES
+            connHandler = self._TapPlus__getconnhandler()
+            response = connHandler.execute_tapget(subContext, False)
+            if response.status == 200:
+                for line in response:
+                    string_message = line.decode("utf-8")
+                    print(string_message[string_message.index('=') + 1:])
+        except OSError as e:
+            print("Status messages could not be retrieved")
```

Comment:
Why the try/except, why not raise this as an exception instead? Is it expected to be unstable?

Also, no need for catching the exception (`as e`) if you don't use it later.

Reply:
updated. Thank you!